apis/kueue/v1beta1/workload_types.go

```diff
@@ -374,6 +378,10 @@ const (
 	// WorkloadLocalQueueRestarted indicates that the workload was requeued because
 	// local queue was restarted after being stopped.
 	WorkloadLocalQueueRestarted = "LocalQueueRestarted"
+
+	// WorkloadRequeuingBackoffLimitExceeded indicates that the workload exceeded max number
+	// of re-queuing retries.
+	WorkloadRequeuingBackoffLimitExceeded = "RequeuingBackoffLimitExceeded"
```

Comment:
```suggestion
	WorkloadRequeuingLimitExceeded = "RequeuingLimitExceeded"
```
and the comment above

Reply:
Fixed